bundle/config/resources/registered_model.go

```diff
@@ -0,0 +1,25 @@
+package resources
+
+import (
+	"github.com/databricks/cli/bundle/config/paths"
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+)
+
+type RegisteredModel struct {
+	// This is a resource agnostic implementation of grants for ACLs.
+	// Implementation could be different based on the resource type.
```

Comment:
"grants for ACLs" reads weird, leftover from copy/paste (I think).

Reply:
Fixed!